lib/android/app/src/main/res/values/dimen.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="margin">16dp</dimen>
```

Comment:
why do we need it in xml? move it to a constant where it's used

Reply:
if it's constant, you need to handle converting dp to px. It's just easier with .xml

Reply:
I don't think it's a good idea to put such a general constant in the resources. which margin? where? also, how are the res being merged with RN apps? did you check?